android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -68,6 +78,88 @@
   Map<String, Object> constants;
   AsyncTask<Void, Void, Map<String, Object>> futureConstants;
 
+  private boolean isDebug = true;
+
+  private boolean isTelephony = true;
+
+  private boolean isCheckPackage = true;
+
+  private List<String> mListPackageName = new ArrayList<>();
+
+  private static final String IP = "10.0.2.15";
+
+  private static final int MIN_PROPERTIES_THRESHOLD = 0x5;
```

Comment:
This should have a javadoc comment I think? I'm not sure why 0x5 is the difference between 'properties' (what kind of properties?) being above or below a value that means anything by just looking at it

Reply:
@mikehardy I'm not sure if this should have javadoc comment... besides I'm wondering if we should remove the whole checkQEmuProps. After review I'm quite sure that is redundant,  though the method check some properties related to CheckBuildInfo as `Build.HARDWARE.equals("goldfish") `

checkQEmuProps has properties checker as bellow:
`        new Property("ro.bootloader", "unknown"), `
`        new Property("ro.hardware", "goldfish"), `  

It's a more specific checking, searching for buildInfo in a rooted android device, should I consider remove? About 0x5 was a mistake already fixed, **MIN_QEMU_PROPERTIES_THRESHOLD** above 5 I consider to be an emulated device.

Reply:
Any time we can remove "magic numbers" from code I usually like it - "constants" have a habit of changing over time, and the less future maintenance the better

Reply:
As I said I'll remove this method... with all others methods already implemented is this PR, we should be able to block those Android emulators as NoxPlayer, BlueStacks, AndyEmulator, etc...